modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/workflow-definitions/underwriting/workflow-definition.xml

```diff
@@ -0,0 +1,324 @@
+
+import java.io.Serializable;
+import java.util.Map;
+
+long classPK = GetterUtil.getLong((String)workflowContext.get(
+WorkflowConstants.CONTEXT_ENTRY_CLASS_PK));
+
+ObjectEntry objectEntry = ObjectEntryLocalServiceUtil.getObjectEntry(classPK);
+
+Map<String, Serializable> values = objectEntry.getValues();
+
+Serializable applicationStatus = values.get("applicationStatus");
+
+String status = applicationStatus.toString();
+
+if(status.equals("underwriting")){
+returnValue = "yes";
+}else {
+returnValue = "no";
+}]]>
+		</script>
+		<script-language>groovy</script-language>
+		<transitions>
+			<transition>
+				<labels>
+					<label language-id="en_US">
+						yes
+					</label>
+				</labels>
+				<name>yes</name>
+				<target>underwriter</target>
+				<default>true</default>
+			</transition>
+			<transition>
+				<labels>
+					<label language-id="en_US">
+						no
+					</label>
+				</labels>
+				<name>no</name>
+				<target>719cecc5-6b67-4b4d-a182-dfba55527a60</target>
+				<default>false</default>
+			</transition>
+		</transitions>
+	</condition>
+	<state>
+		<name>created</name>
+		<description>Begin a workflow.</description>
+		<metadata>
+			<![CDATA[
+				{
+					"xy": [
+						282.5476078091601,
+						127.42522141700385
+					]
+				}
+			]]>
+		</metadata>
+		<initial>true</initial>
+		<labels>
+			<label language-id="en_US">
+				created
+			</label>
+		</labels>
+		<transitions>
+			<transition>
+				<labels>
+					<label language-id="en_US">
+						Transition Label
+					</label>
+				</labels>
+				<name>38f5d06e-2e1c-453c-bf02-5f06899bf4ef</name>
+				<target>9247483e-463b-4f58-8482-f524cf1a50e8</target>
+				<default>true</default>
+			</transition>
+		</transitions>
+	</state>
+	<state>
+		<name>reviewed</name>
+		<description>Conclude the workflow.</description>
+		<metadata>
+			<![CDATA[
+				{
+					"terminal": true,
+					"xy": [
+						752.589984675345,
+						664.6855668823064
+					]
+				}
+			]]>
+		</metadata>
+		<actions>
+			<action>
+				<name>reviewed</name>
+				<description></description>
+				<script>
+							<![CDATA[import com.liferay.portal.kernel.service.UserLocalServiceUtil;
+import com.liferay.object.model.ObjectEntry;
+import com.liferay.object.service.ObjectEntryLocalServiceUtil;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.workflow.WorkflowConstants;
+import com.liferay.portal.kernel.workflow.WorkflowStatusManagerUtil;
+
+import java.io.Serializable;
+import java.util.Map;
+
+	long classPK = GetterUtil.getLong((String)workflowContext.get(
+	WorkflowConstants.CONTEXT_ENTRY_CLASS_PK));
+
+	ObjectEntry objectEntry = ObjectEntryLocalServiceUtil.getObjectEntry(classPK);
+
+	Map<String, Serializable> values = objectEntry.getValues();
+
+	values.put("applicationStatus", "reviewed");
+
+	long companyId = serviceContext.getCompanyId();
+
+	long userId = UserLocalServiceUtil.getUserIdByEmailAddress(companyId, "ryan.underwriter@mailinator.com");
```

Comment:
You can use companyId as inline and the same for the userId in the next line

Reply:
Done